app/models/game.rb

```diff
@@ -53,6 +53,15 @@ def not_your_turn
     !inactive_player
   end
 
+  def forfeit_game(current_player_id, game_id)
```

Comment:
Can pass objects here instead of ID's. Have already found these records once in the controller, no need to run another query to look them up again